Sources/Sentry/SentryTimeToDisplayTracker.m

```diff
@@ -140,6 +142,9 @@ - (void)framesTrackerHasNewFrame:(NSDate *)newFrameDate
         if (!_waitForFullDisplay) {
             [SentryDependencyContainer.sharedInstance.framesTracker removeListener:self];
 #    if SENTRY_TARGET_PROFILING_SUPPORTED
+            if (SentrySDK.options.enableContinuousProfiling) {
+                return;
+            }
             sentry_stopAndDiscardLaunchProfileTracer();
```

Comment:
`h`: I think we must not return here and also not below, as it could happen that with a new frame both the TTFD and TTID spans need to be finished. It would be great to add test for this. Instead it should be:
```suggestion
            if (!SentrySDK.options.enableContinuousProfiling) {
                sentry_stopAndDiscardLaunchProfileTracer();
            }

```

Reply:
did this one and the other one